api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -287,6 +287,8 @@ message HttpConnectionManager {
   // is not desired it can be disabled.
   google.protobuf.BoolValue generate_request_id = 15;
 
+  google.protobuf.BoolValue edge_accept_request_id = 32;
```

Comment:
The field should probably have a descriptive comment, as the user documentation is rendered from these comments.

Reply:
done